<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Archiver</class>
 <widget class="QMainWindow" name="Archiver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Archiver</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="b_create">
        <property name="text">
         <string>Create archive</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="b_open">
        <property name="text">
         <string>Open archive</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="b_add">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Add files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="b_delete">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Delete files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="b_extract">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Extract files</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="l_files"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
